lib/components/narrative/line-itin/itin-summary.tsx

```diff
@@ -209,6 +204,7 @@ export class ItinerarySummary extends Component<Props> {
 
 // Helper functions
 
+// Leg is any for now until types package is updated with correct Leg type
```

Comment:
Are we not able to update this type?

Reply:
We do, but not in this PR! Didn't want to have competing types package releases that make the same changes. I think this change is part of the `core-utils` typescript PR.